source/common/formatter/stream_info_formatter.cc

```diff
@@ -856,6 +1015,11 @@ const StreamInfoFormatterProviderLookupTable& getKnownStreamInfoFormatterProvide
                     return stream_info.currentDuration();
                   });
             }}},
+          {"COMMON_DURATION",
+           {CommandSyntaxChecker::PARAMS_REQUIRED | CommandSyntaxChecker::LENGTH_ALLOWED,
```

Comment:
Sorry, it's been a while since I was in here, but I think `LENGTH_ALLOWED` is probably not warranted here since it would be truncating a number.

Reply:
These flags are used to validate the format of command.  I actually ignored the length.

But you are right, I will remove it.